### PR TITLE
Added Support for G935

### PR DESCRIPTION
**Pull request by gort818**

### CODE REVIEW

**Comment by Sapd** [comment]
Instead of creating new files for G935, you can simply rename the one of G933 to G93X and add the device id to the array on line 21 (like done in corsair_void.c)

**Comment by Sapd** [comment]
The request for the sidetone differs in the implementation. So we can't be sure that the implementation of G933 will work for that too. 
I also don't understand why so many Logitech devices differ in protocol, whereas Corsair always seem to use the same.

**Comment by gort818** [comment]
That is what I am saying if we use G93X .. that would mean all  G93X devices which could be confused as G930.

But I can do that no problem if that works for you.

**Comment by Sapd** [comment]
Ah didn't thought about that, another suggestion is simply logitech_g933_935.c

**Comment by gort818** [comment]
Ok check it out

File: src/device.h [hunk 1]
```diff
@@ -20,7 +20,7 @@ enum capabilities {
 /** @brief Flags for battery status
  */
 enum battery_status {
-    BATTERY_LOADING         = 65535
+    BATTERY_CHARGING         = 65535
```
Comment (Sapd) [inline src/device.h]: Such changes are better done in a seperate commit/PR

File: src/devices/logitech_g933_935.c [hunk 2]
```diff
@@ -5,31 +5,35 @@
 #include <stdlib.h>
 #include <hidapi.h>
 
-static struct device device_g933;
+static struct device device_void;
```
Comment (Sapd) [inline src/devices/logitech_g933_935.c]: Note you named it "void"

File: src/devices/logitech_g933_935.c [hunk 3]
```diff
@@ -5,31 +5,35 @@
...
+static const uint16_t PRODUCT_IDS[] = {ID_LOGITECH_G933,ID_LOGITECH_G935};
 
-void g933_init(struct device ** device)
-{ 
-    device_g933.idVendor = VENDOR_LOGITECH;
-    device_g933.idProductsSupported = &PRODUCT_ID;
-    device_g933.numIdProducts = 1;
+static int void_send_sidetone(hid_device *device_handle, uint8_t num);
+static int void_request_battery(hid_device *device_handle);
+static int void_lights(hid_device *device_handle, uint8_t on);
```
Comment (Sapd) [inline src/devices/logitech_g933_935.c]: Also here you named it void

File: src/device.h (hunk 1 above)
Comment (Sapd) [reply src/device.h]: Ah I just saw that it's done seperatly and the rest is squashed. Then it's okay in this case

**Comment by gort818** [comment]
oops. fixed

**Pull request merged**